floodlight/io/sportradar.py

```diff
@@ -0,0 +1,199 @@
+from pathlib import Path
+from typing import Dict, Union
+
+import datetime
+import json
+import pandas as pd
+
+from floodlight import Events
+
+
+def read_sportradar_timeline(
+    filepath_events: Union[str, Path]
+) -> Dict[str, Dict[str, Dict]]:
+    """Parses the sport radar timeline files in json format and extracts the event data.
+
+    This function provides access to the `Sport Event Timeline
+    <https://developer.sportradar.com/docs/read/handball/
+    Handball_v2#sport-event-timeline>`_ file from the dataprovider `Sportradar
+    <https://sportradar.com/>`_ exported in json format and returns Event objects for
+    all teams and segments of the game.
+
+    Parameters
+    ----------
+    filepath_events: str or pathlib.Path
+        Full path to json file where the Sport Event Timeline is saved.
+
+    Returns
+    -------
+    data_objects: Dict[str: Dict[str: Events]]
+        Dict with Events-objects for all teams and segments. For a usual league match
+        with two halves and two teams the order is:
+
+        {
+
+            1st_half: {team_a: Events, team_b: Events},
+
+            2nd_half: {team_a: Events, team_b: Events}
+
+        }
```

Comment:
Thanks and great additions. I just put all the highlighted words in double backticks. I think that is consistent with the rest of the docs.

On your note: It was a semi-deliberate mishap just following the statsbomb parser but knowing this would probably be changed one way or another ;) I now changed it to ["HT1", "HT2"] and ["Home", "Away"]. We will if this solution will work out smoothly with matches with possible overtime.